### PR TITLE
1818932: 1820267: Using 'Simple Content Access' for access mode

### PR DESCRIPTION
**Pull request by wottop**

### CODE REVIEW

**Comment by wottop** [comment]
buildbot: retest this please

File: src/subscription_manager/gui/registergui.py [hunk 1]
```diff
@@ -824,7 +824,7 @@ def __init__(self, backend, reg_info=None,
                                              parent_window)
 
     def choose_initial_screen(self):
-        if is_owner_using_golden_ticket():
+        if is_simple_content_access():
             log.debug('Skipping auto-bind, because contentAccessMode is equal to organization')
```
Comment (cnsnyder) [inline src/subscription_manager/gui/registergui.py:828]: We should update this message as well.

File: src/subscription_manager/utils.py [hunk 2]
```diff
@@ -161,7 +161,7 @@ def is_valid_server_info(conn):
         return False
 
 
-def is_owner_using_golden_ticket(uep=None, identity=None, owner=None):
+def is_simple_content_access(uep=None, identity=None, owner=None):
     """
     This function return True, when current owner uses contentAccessMode equal to org_environment. This
```
Comment (cnsnyder) [inline src/subscription_manager/utils.py:166]: Is the value of the contentAccessMode still "org_environment"?

Comment (wottop) [reply src/subscription_manager/utils.py:166]: Yes

**Comment by cnsnyder** [comment]
@mvollmer Hi! Is there anything we can do in this repo to help get your RHEL-8-2 test to pass?

**Pull request merged**